mne/coreg.py

```diff
@@ -1207,3 +1207,44 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
                 F_mri_ras, 'ras', 'ras'),
             F_ras_mni, 'ras', 'mni_tal')
     _write_fs_xfm(fname_to, T_ras_mni['trans'], kind)
+
+
+def get_mni_fiducials(subject, subjects_dir=None):
+    """Estimate fiducials for a subject.
```

Comment:
okay I am probably asking my old question again but rephrased, so sorry for my insistence.

how exactly do you "estimate"? Can you write that in words? Since this is a public function, I'd expect some information about how this is done in the docstring.

Reply:
description added